catalyst/dl/callbacks/checkpoint.py

```diff
@@ -77,8 +84,12 @@ def save_checkpoint(
             last_filepath = last_item[0]
             os.remove(last_filepath)
 
-    def pack_checkpoint(self, **kwargs):
-        return utils.pack_checkpoint(**kwargs)
+        checkpoints = [
+            (Path(filepath).stem, metric)
+            for (filepath, _, metric) in self.top_best_metrics
+        ]
+        metrics = OrderedDict(checkpoints + [("last", valid_metrics)])
```

Comment:
maybe you should also save "best" checkpoint metrics? so it would be the same as checkpoints naming